// Copyright (c) 2020-2021 The Open-Transactions developers
// This Source Code Form is subject to the terms of the Mozilla Public
// License, v. 2.0. If a copy of the MPL was not distributed with this
// file, You can obtain one at http://mozilla.org/MPL/2.0/.

syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTXEnums";
option optimize_for = LITE_RUNTIME;

enum ServerRequestType {
    SERVERREQUEST_ERROR = 0;
    SERVERREQUEST_ACTIVATE = 1;
}

enum ServerReplyType {
    SERVERREPLY_ERROR = 0;
    SERVERREPLY_ACTIVATE = 1;
    SERVERREPLY_PUSH = 255;
}

enum OTXPushType {
    OTXPUSH_ERROR = 0;
    OTXPUSH_NYMBOX = 1;
    OTXPUSH_INBOX = 2;
    OTXPUSH_OUTBOX = 3;
}
